frontend/src/components/nodes/ValidatorTelemetryRow.tsx

```diff
@@ -58,10 +64,26 @@ const ValidatorTelemetryRow: FC<Props> = ({
               expectedBlocks !== undefined &&
               expectedBlocks !== 0 ? (
                 <>
-                  {((producedBlocks / expectedBlocks) * 100).toFixed(3)}% &nbsp;
-                  <span>
-                    ({producedBlocks}/{expectedBlocks})
-                  </span>
+                  <OverlayTrigger
+                    placement={"bottom"}
+                    overlay={
+                      <Tooltip id="produced-blocks-chunks">
+                        {t(
+                          "component.nodes.ValidatorTelemetryRow.produced_blocks_and_chunks",
+                          {
```

Comment:
Could it be that we have amount of blocks, but not amount of chunks?
Should we put something like `num_produced_chunks: producedChunks ?? t('..unknown')`?

Reply:
Nope, either we have all 4 values or none of them. I would consider having a structure with all 4 values required and make that structure optional to make this relation crystal clear @shelegdmitriy